### DISCUSSION TITLE
Clean up individual drum tracks from live recording

### DISCUSSION THREAD

**Discussion by martinstender** [Anjok07/ultimatevocalremovergui#940] [General]
Hi all,

I have a multitrack from a show we did, where we might end up using some of the drums and lead vocals on a future release.
But there's a lot of bleed in those drum mics. Especially the overheads, of course, but generally, it would be great to clean it up as much as possible.
I know its a lot to ask, but has anyone come across a good way to remove certain elements from a drum kit, while leaving, say, the overheads, or the snare intact? 

Using the 'default' v4 htdemucs, isn't bad at all. It gets the vocals out of the snare mic for example, but of course the cymbals still bleeds into that mic. Generally, the 6 drum mics/tracks, with the exception of the kick, contains a lot of the other kit elements.

Any suggestions?

Thanks in advance :)

**Comment by ghost** [comment]
Take a look at DrumSep, this is a Demucs model trained to separated individual drums elements (kick/snare/toms/cymbals)
There's some explanation to get the models and to use it here https://github.com/facebookresearch/demucs/issues/422#issuecomment-1501932895  and here https://github.com/inagoy/drumsep/issues/3

**Reply by martinstender** [reply]
Many thanks 😊🙏
